ADApp/pluginSrc/NDPluginDriver.cpp

```diff
@@ -72,6 +72,10 @@ static const char *driverName="NDPluginDriver";
     if (dimsChanged) {
         doCallbacksInt32Array(this->dimsPrev, ND_ARRAY_MAX_DIMS, NDDimensions, 0);
     }
+    // Save a pointer to the input array for use by ProcessPlugin
```

Comment:
The plugins now hang on to the last received NDArray - potentially forever. Would it be useful to have feature to force the plugin to drop it? 

It could for example be dropped when the plugin is disabled?